README.md

```diff
@@ -76,9 +75,11 @@ pip install -e .
 
 - `PUDL_DB=YOUR_PATH_HERE` (your path to the PUDL database downloaded in step 5)
 - `PG_DB=YOUR_PATH_HERE` (your path to the additional PowerGenome data downloaded in step 6)
-- `RESOURCE_GROUP_PROFILES=YOUR_PATH_HERE` (your path to the folder with hourly wind/solar generation parquet files)
-- `EFS_DATA=YOUR_PATH_HERE` (your path to the folder with EFS derived data files)
-- `DISTRIBUTED_GEN_DATA=YOUR_PATH_HERE` (your path to the folder with distributed generation profiles)
+- `RESOURCE_GROUP_PROFILES=YOUR_PATH_HERE` (your path to the folder with hourly wind/solar
```

Comment:
@gschivley is `RESOURCE_GROUP_PROFILES` superseded by `RESOURCE_GROUPS`? (I'm working with some older PowerGenome inputs that only had the option of `RESOURCE_GROUPS` in the .env file - so the profiles are mapped via the JSON files with `"profiles"` value)

Reply:
If `RESOURCE_GROUP_PROFILES` exists, PG will look there for the profiles. If it doesn't, the fallback location is `RESOURCE_GROUPS`.

Reply:
ok can you please merge these changes or suggest changes that you'd like to see?